docs/guide.md

```diff
@@ -0,0 +1,575 @@
+# Firebase Security and Rules Using the Bolt Compiler
+
+Firebase is secured using a JSON-formatted [Security and
+Rules](https://www.firebase.com/docs/security/guide/understanding-security.html) language. It
+is a powerful feature of Firebase, but can be error prone to write by hand.
+
+The Bolt compiler helps developers express the schema and authorization rules for their
+database using a familiar JavaScript-like language. The complete [language
+reference](language.md) describes the syntax. This guide introduces the concepts and
+features of Bolt along with a cookbook of common recipies.
+
+# Getting Started
+
+The Firebase Bolt compiler is a command-line based tool, written in node.js. You can install it
+using the [Node Package Manager](https://nodejs.org/en/download/):
+
+    $ npm install --global firebase-bolt
+
+You can use the Bolt compiler to compile the examples in this guide, and inspect the output
+JSON.
+
+## Default Firebase Permissions
+
+By default, Firebase has a permissive security model - this makes it easy to test your code
```

Comment:
By default, Firebase has a permissive security model: everyone can read and write all data. This makes it easy to test your code, but is unsafe for production apps since anyone can read and overwrite all your data.